src/components/content/content.pw.tsx

```diff
@@ -0,0 +1,129 @@
+import React from "react";
+import { test, expect } from "@playwright/experimental-ct-react17";
+import ContentComponent from "./components.test-pw";
+import { CHARACTERS } from "../../../cypress/support/component-helper/constants";
+import {
+  contentBody,
+  contentTitle,
+} from "../../../playwright/components/content";
+import {
+  checkAccessibility,
+  getStyle,
+} from "../../../playwright/support/helper";
+
+const testData = [CHARACTERS.DIACRITICS, CHARACTERS.SPECIALCHARACTERS];
+const totalWidth = 1366;
+const alignmentData = ["left", "center", "right"];
+
+test.describe("check Content component properties", () => {
+  [
+    ["primary", "rgba(0, 0, 0, 0.9)"],
+    ["secondary", "rgba(0, 0, 0, 0.55)"],
+  ].forEach(([variant, titleColor]) => {
+    test(`should check ${variant} as variant for Content components`, async ({
+      mount,
+      page,
+    }) => {
+      await mount(<ContentComponent variant={variant} />);
+      const contentElement = contentTitle(page);
```

Comment:
**suggestion**: the `contentElement`, `const` doesn't need to be declared here. You can just include the `contentTitle` locator within the `expect` underneath, reducing a line of code like so `await expect(contentTitle(page)).toHaveCSS("color", `${titleColor}`);`